.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Feature Request
+description: Used to report a feature request for AMP.
+labels: "Type: Feature Request"
+body:
+- type: markdown
+  id: header
+  attributes:
+    value: |
+      Thanks for filling out this feature request.
+      - Feature requests related to the [AMP](https://amp.dev) format and cache can be reported using the form below.
+      - Feature requests related to the [AMP Wordpress Plugin](https://wordpress.org/plugins/accelerated-mobile-pages/) can be reported at the [`amp-wp`](https://github.com/ampproject/amp-wp/issues) repository.
```

Comment:
Follow-up to improve WordPress references: https://github.com/ampproject/amphtml/pull/34401

Reply:
Excellent catches, thanks for noticing them and for sending out the fix. 👍